docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -187,7 +187,7 @@ Developers can use the `DECRYPT` permission to decrypt data or messages, for ins
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000000800</code>
     </p>
 
-Developers can use the `SIGN` permission for keys to sign login messages. It is primarily for web2.0 apps to know which key SHOULD sign.
+The permission `SIGN` enables an address to authenticate on behalf of the UP. It can be used primarily in web2.0 apps to sign login messages.
```

Comment:
```suggestion
The permission `SIGN` enables an address to authenticate on behalf of the UP. It can be used primarily in web2.0 apps to [sign login messages](-lets put a link here to the SIWE guide - relative link-).
```

Reply:
i added it in the next commit